scripts/fetch:deployment.ts

```diff
@@ -10,7 +10,7 @@ type FetchDetails = {
 }
 const {
   nexusDeploymentPath = "../node_modules/nexus/deployments",
-  chainName = "anvil-55000",
+  chainName = "anvil-51654",
```

Comment:
@joepegler perhaps has to do with this?

Reply:
No this only runs when you manually run it.